library/forms/lib/fieldRegistry/fields/MapField/index.tsx

```diff
@@ -240,7 +241,42 @@ export function MapFormField(props: FieldProps): JSX.Element {
             gap: theme.spacing(1),
           }}
         >
-          <MapPreview value={props.state.value?.data} config={mapConfig} />
+          {props.config.platform === 'web' && (
```

Comment:
I don't think we should have a different UI for web here - make it consistent on all platforms.